### ISSUE TITLE
Feature Idea: London postcodes in sample data

### ISSUE THREAD

**Issue by ciacicode** [bokeh/bokeh#2487]
While experimenting with bokeh I realised that it does not have data to draw a map outside the US. I thought it would be nice to have London postcode data so I have collected and manipulated polygon data for the postcode macro areas of the London, UK.

I forked the project and wanted to know if you as a team are interested in this sample data. The output of the module is a set of dicts as follows:
```
data[(parent_id,area_id)]['name']
data[(parent_id,area_id)]['lats']
data[(parent_id,area_id)]['lons']
```

You can see my commit in the "2487-london-sample-data" branch [here](https://github.com/ciacicode/bokeh/commit/af05a7c19b2ea9d1b87374f99834bacc168191bb). 

The change includes:
* A new csv file
* A new module to extract the data dict with the geometry data

**Comment by ciacicode** [comment]
Shall I try sending directly the pull request?

**Comment by damianavila** [comment]
PRs are always welcome :wink:

**Comment by ciacicode** [comment]
Hello @damianavila , I kept on working on this after closing the pull request. My problem is expressed on [Google Group](https://groups.google.com/a/continuum.io/forum/#!searchin/bokeh/london/bokeh/iKBPrpsqKso/Ou5jbFk4DcgJ), where I link to the repository branch where I am building the map (it's a flask application but the only files of interest are static/london_fci.html , london_postcodes.py and charts.py

TLDR:
The longitudes of London are very small, very close to 0 for the obvious proximity to Greenwich, I find that all my longitudes are being converted from floats to exponential numbers, causing some postcodes to appear in totally wrong longitude. 

Any help, advise is greatly welcome!

**Comment by bryevdv** [comment]
Closing for inactivity.
